### PR TITLE
#2 database models

### PR DESCRIPTION
**Pull request by tombuckley91**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "A homework tracking software",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (rhysb27) [inline package.json:7]: No tests or test script? Unnacceptable.

**Review by rhysb27 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
...
+    "start": "node server.js"
```
Comment (rhysb27) [inline package.json:8]: Please add a test script.

**Review by rhysb27 — CHANGES_REQUESTED** [review]

**Review by rhysb27 — APPROVED** [review]
LGTM!

File: package.json (hunk 2 above)
Comment (tombuckley91) [reply package.json:8]: Will add tests when appropriate.

**Pull request merged**